hack/util.sh

```diff
@@ -458,10 +458,14 @@ function install_registry() {
 }
 readonly -f install_registry
 
-function wait_for_registry() {
+function wait_generation() {
```

Comment:
Maybe I should rename it to `is_registry_ready`? Because this function doesn't wait but returns the status of deployment.